states/base.py

```diff
@@ -15,6 +15,15 @@ def __init__(self):
         self.killed = False
         self.on_pi = which("rpi-update")
 
+    def add_player(self,player):
```

Comment:
```suggestion
    def player_added(self,player):
```
Om duidelijker te maken dat dit een callback is, geen method om een player toe te voegen